docs-src/content/using-cubic-sdk/client-configs.md

```diff
@@ -0,0 +1,108 @@
+}
+```
+
+and your resulting transcript would be found at `results.alternatives[*].transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Sometimes, only the Raw transcript is desired.  In this case, there are two options.  
+
+1. Disable server-side formatting and use the above config.  Retrieve the raw transcript from the `results.alternatives[*].transcript` field.
+2. Specify the `enable_raw_transcript = true` flag, and access the field `results.alternatives[*].raw_transcript`.
+
+> Note: Prior to `cubicsvr` v2.9.0 and SDK-Cubic v1.3.0, the field `results.alternatives[*].transcript` was populated with either the raw or the formatted transcription depending on the `enable_raw_transcript` config.  After these changes, raw transcripts have been pushed to a new field `results.alternatives[*].raw_transcript`, and only populated when `enable_raw_transcript` is set to true.
+
+------
+
+Another use case would be getting both the formatted and raw transcript.  This can be done using this config.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": "true"
+}
+```
+
+The formatted transcript would be found at `results.alternatives[*].transcript`, and the raw transcript would be found at `results.alternatives[*].raw_transcript`.
+
+------
+
+If you needed to dive deeper in word level timestamps, say for subtitles, then you can use this config to enable those word-level timestamps.  Please note the inclusion of `enable_raw_transcript`.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": true,
+    "enable_word_time_offsets": true
+}
+```
+
+Timestamps will be found at `results.alternatives[*].words[*].start_time` and `results.alternatives[*].words[*].duration`.
+
+------
+
+If you needed to dive deeper in word level confidences, i.e. for displaying a lighter color for less confident words, then you can use this config to enable those word-level confidence values.  Please note the inclusion of `enable_raw_transcript`.
+
+```json
+{
+    "model_id": "1",
+    "audio_encoding": "raw",
+    "enable_raw_transcript": true,
+    "enable_word_time_offsets": true
+}
+```
+
+Word-level confidence scores will be found at `results.alternatives[*].words[*].confidence`.
+ 
+------
+
+For power users, such as those combining results from multiple ASR engines, the most comprehensive and detailed Cubic results are found in the confusion network.  Please refer to the [in depth confusion network documetation](../../protobuf/autogen-doc-cubic-proto/#message-recognitionconfusionnetwork) for more details.
```

Comment:
Also, I think it would make more sense to link to the confusion network documentation on SDK website homepage (there isn't much of a description in the protobuf documentation). I *think* the hyperlink is something like `../../#confusion-network`, but I'm not entirely sure how Hugo translates these from markdown.